src/Microsoft.AspNet.Owin/project.json

```diff
@@ -10,7 +10,8 @@
     "keyFile": "../../tools/Key.snk"
   },
   "dependencies": {
-    "Microsoft.AspNet.Http": "1.0.0-*"
+    "Microsoft.AspNet.Http": "1.0.0-*",
+    "Microsoft.Extensions.DependencyInjection": "1.0.0-*"
```

Comment:
We need the implementation `ServiceCollection` from `DependencyInjection`. That is not available in `DependencyInject.Abstractions`.